lib/client/jsx/components/manifest/consignment_view.jsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import ConsignmentResult from './consignment_result';
+
+export default class ConsignmentView extends React.Component {
+  render() {
+    let { consignment } = this.props;
+
+    if (!consignment) return null;
+
+    console.log("Consignment is ");
+    console.log(consignment);
+
+    return <div className='consignment-view'>
+      <div className='label'>Results:</div>
+      {
+        Object.keys(consignment).map(
+          (name,i) => {
+            if (consignment[name] == 'macro') return null;
```

Comment:
Whatever 'macro' consignments are should probably be removed before getting to the "ConsignmentView"